### PR TITLE
Generically interleave hashconsing with hash computation

### PR DESCRIPTION
**Pull request by ppedrot**
This allows to have a better algorithmic complexity for recursive datatypes.

### CODE REVIEW

File: kernel/names.ml [hunk 1]
```diff
@@ -551,24 +570,28 @@ module KerPair = struct
...
       let hashcons hkn = function
-        | Same kn -> Same (hkn kn)
-        | Dual (knu,knc) -> make (hkn knu) (hkn knc)
+        | Same kn ->
+          let kn, h = hkn kn in
+          Same kn, h
+        | Dual (knu, knc) ->
+          let knu, hu = hkn knu in
+          let knc, hc = hkn knc in
+          if KerName.equal knu knc then
```
Comment (SkySkimmer) [inline kernel/names.ml:585]: Since this is immediately after hashconsing could we use physical equality?

Comment (ppedrot) [reply kernel/names.ml:585]: ... maybe. For now I just want to bench this to see if this rough POC has any observable effect (which I hope).

**Comment by ppedrot** [comment]
https://gitlab.com/coq/coq/-/jobs/756080440
```
┌────────────────────────────┬─────────────────────────┬─────────────────────────────────────────────┬─────────────────────────────────────────────┬────────────────────────────────┬───────────────────┐
│                            │      user time [s]      │                 CPU cycles                  │              CPU instructions               │     max resident mem [KB]      │    mem faults     │
│                            │                         │                                             │                                             │                                │                   │
│               package_name │     NEW     OLD PDIFF   │               NEW               OLD PDIFF   │               NEW               OLD PDIFF   │        NEW        OLD  PDIFF   │  NEW  OLD PDIFF   │
├────────────────────────────┼─────────────────────────┼─────────────────────────────────────────────┼─────────────────────────────────────────────┼────────────────────────────────┼───────────────────┤
│                coq-bignums │   57.69   58.05 -0.62 % │      159593590346      159443425973 +0.09 % │      214715357449      215213255122 -0.23 % │     472968     471980  +0.21 % │    0    0  +nan % │
├────────────────────────────┼─────────────────────────┼─────────────────────────────────────────────┼─────────────────────────────────────────────┼────────────────────────────────┼───────────────────┤
│               coq-compcert │  659.70  662.31 -0.39 % │     1836851794176     1844764037413 -0.43 % │     2535586883279     2553714764155 -0.71 % │    1129840    1130408  -0.05 % │    0    0  +nan % │
├────────────────────────────┼─────────────────────────┼─────────────────────────────────────────────┼─────────────────────────────────────────────┼────────────────────────────────┼───────────────────┤
│            coq-lambda-rust │ 1250.00 1254.07 -0.32 % │     3478076820701     3491172128613 -0.38 % │     4863958939271     4937343724944 -1.49 % │    1168320    1166372  +0.17 % │    0    0  +nan % │
├────────────────────────────┼─────────────────────────┼─────────────────────────────────────────────┼─────────────────────────────────────────────┼────────────────────────────────┼───────────────────┤
│     coq-mathcomp-character │  159.74  160.15 -0.26 % │      444443971133      444770290068 -0.07 % │      612793464222      617835699278 -0.82 % │     719628     720476  -0.12 % │    0    0  +nan % │
├────────────────────────────┼─────────────────────────┼─────────────────────────────────────────────┼─────────────────────────────────────────────┼────────────────────────────────┼───────────────────┤
│     coq-mathcomp-ssreflect │   50.22   50.34 -0.24 % │      137611052026      138022916260 -0.30 % │      174939590830      176121198314 -0.67 % │     523684     521396  +0.44 % │    0    0  +nan % │
├────────────────────────────┼─────────────────────────┼─────────────────────────────────────────────┼─────────────────────────────────────────────┼────────────────────────────────┼───────────────────┤
│               coq-coqprime │   80.23   80.39 -0.20 % │      220832578514      221039088772 -0.09 % │      309159490192      310595649196 -0.46 % │     775984     775868  +0.01 % │    0    0  +nan % │
├────────────────────────────┼─────────────────────────┼─────────────────────────────────────────────┼─────────────────────────────────────────────┼────────────────────────────────┼───────────────────┤
│      coq-mathcomp-fingroup │   47.81   47.85 -0.08 % │      132848678970      132753482413 +0.07 % │      182165168190      183675778919 -0.82 % │     486188     486344  -0.03 % │    0    0  +nan % │
├────────────────────────────┼─────────────────────────┼─────────────────────────────────────────────┼─────────────────────────────────────────────┼────────────────────────────────┼───────────────────┤
│                        coq │  528.74  529.05 -0.06 % │     1477601747005     1478462975971 -0.06 % │     1986704751151     1994983532272 -0.41 % │     591452     596696  -0.88 % │    0    0  +nan % │
├────────────────────────────┼─────────────────────────┼─────────────────────────────────────────────┼─────────────────────────────────────────────┼────────────────────────────────┼───────────────────┤
│           coq-fiat-parsers │  646.75  647.06 -0.05 % │     1810402677088     1812191000122 -0.10 % │     2711075912960     2734568554810 -0.86 % │    2997056    3420312 -12.37 % │    0    0  +nan % │
├────────────────────────────┼─────────────────────────┼─────────────────────────────────────────────┼─────────────────────────────────────────────┼────────────────────────────────┼───────────────────┤
│                 coq-geocoq │ 1466.13 1466.76 -0.04 % │     4084077742575     4088368418597 -0.10 % │     6201614182794     6214956781662 -0.21 % │    1146036    1149644  -0.31 % │    0    0  +nan % │
├────────────────────────────┼─────────────────────────┼─────────────────────────────────────────────┼─────────────────────────────────────────────┼────────────────────────────────┼───────────────────┤
│                   coq-hott │  322.73  322.67 +0.02 % │      877645548459      876839491093 +0.09 % │     1412238823901     1427183043813 -1.05 % │     634128     733960 -13.60 % │    0    0  +nan % │
├────────────────────────────┼─────────────────────────┼─────────────────────────────────────────────┼─────────────────────────────────────────────┼────────────────────────────────┼───────────────────┤
│            coq-fiat-crypto │ 4980.46 4979.40 +0.02 % │    13828566103015    13831341661391 -0.02 % │    23873931623344    23986584630305 -0.47 % │    2381476    2392128  -0.45 % │    0    0  +nan % │
├────────────────────────────┼─────────────────────────┼─────────────────────────────────────────────┼─────────────────────────────────────────────┼────────────────────────────────┼───────────────────┤
│      coq-mathcomp-solvable │  171.88  171.73 +0.09 % │      477810936949      478412222537 -0.13 % │      669825826376      675987559464 -0.91 % │     691752     691644  +0.02 % │    0    0  +nan % │
├────────────────────────────┼─────────────────────────┼─────────────────────────────────────────────┼─────────────────────────────────────────────┼────────────────────────────────┼───────────────────┤
│                coq-unimath │ 4000.94 3997.15 +0.09 % │    11133451873088    11119414050676 +0.13 % │    20948832995952    21169216243691 -1.04 % │    1124396    1110396  +1.26 % │    0    0  +nan % │
├────────────────────────────┼─────────────────────────┼─────────────────────────────────────────────┼─────────────────────────────────────────────┼────────────────────────────────┼───────────────────┤
│           coq-math-classes │  189.36  189.18 +0.10 % │      527808471941      526866883372 +0.18 % │      696348923293      697625925804 -0.18 % │     504756     504496  +0.05 % │    0    0  +nan % │
├────────────────────────────┼─────────────────────────┼─────────────────────────────────────────────┼─────────────────────────────────────────────┼────────────────────────────────┼───────────────────┤
│               coq-rewriter │  608.12  607.43 +0.11 % │     1687671441160     1685284701849 +0.14 % │     2569583867569     2576638009641 -0.27 % │    1037212     993620  +4.39 % │    0    0  +nan % │
├────────────────────────────┼─────────────────────────┼─────────────────────────────────────────────┼─────────────────────────────────────────────┼────────────────────────────────┼───────────────────┤
│                coq-coqutil │   59.13   59.01 +0.20 % │      164205859138      163274614600 +0.57 % │      203976464398      204210752193 -0.11 % │     529604     530768  -0.22 % │    0    0  +nan % │
├────────────────────────────┼─────────────────────────┼─────────────────────────────────────────────┼─────────────────────────────────────────────┼────────────────────────────────┼───────────────────┤
│               coq-bedrock2 │  222.57  222.07 +0.23 % │      616835511991      615536757364 +0.21 % │     1015221167062     1018796575023 -0.35 % │    1087164    1087120  +0.00 % │    0    0  +nan % │
├────────────────────────────┼─────────────────────────┼─────────────────────────────────────────────┼─────────────────────────────────────────────┼────────────────────────────────┼───────────────────┤
│       coq-mathcomp-algebra │  140.22  139.85 +0.26 % │      389438146333      389107424376 +0.08 % │      499524012941      502569607180 -0.61 % │     565880     561712  +0.74 % │    0    0  +nan % │
├────────────────────────────┼─────────────────────────┼─────────────────────────────────────────────┼─────────────────────────────────────────────┼────────────────────────────────┼───────────────────┤
│                  coq-flocq │  158.68  158.12 +0.35 % │      440649475970      439560280734 +0.25 % │      548233445662      551368404081 -0.57 % │     822464     822044  +0.05 % │    0    0  +nan % │
├────────────────────────────┼─────────────────────────┼─────────────────────────────────────────────┼─────────────────────────────────────────────┼────────────────────────────────┼───────────────────┤
│ coq-performance-tests-lite │  546.71  544.49 +0.41 % │     1526702540106     1521564513842 +0.34 % │     2799340517096     2794858254337 +0.16 % │     768320     751056  +2.30 % │    0    0  +nan % │
├────────────────────────────┼─────────────────────────┼─────────────────────────────────────────────┼─────────────────────────────────────────────┼────────────────────────────────┼───────────────────┤
│      coq-engine-bench-lite │  383.04  381.18 +0.49 % │     1076384505974     1072063791180 +0.40 % │     1838608407659     1838479535471 +0.01 % │    3507380    3507260  +0.00 % │    0    0  +nan % │
├────────────────────────────┼─────────────────────────┼─────────────────────────────────────────────┼─────────────────────────────────────────────┼────────────────────────────────┼───────────────────┤
│                   coq-corn │ 1507.53 1499.66 +0.52 % │     4198078775380     4173175898038 +0.60 % │     6370127781968     6395665547879 -0.40 % │     888276     831100  +6.88 % │    0    0  +nan % │
├────────────────────────────┼─────────────────────────┼─────────────────────────────────────────────┼─────────────────────────────────────────────┼────────────────────────────────┼───────────────────┤
│         coq-mathcomp-field │  291.49  289.91 +0.54 % │      810757613032      808246711288 +0.31 % │     1242496061040     1247671510894 -0.41 % │     805516     812004  -0.80 % │    0    0  +nan % │
├────────────────────────────┼─────────────────────────┼─────────────────────────────────────────────┼─────────────────────────────────────────────┼────────────────────────────────┼───────────────────┤
│     coq-mathcomp-odd-order │ 1070.20 1063.26 +0.65 % │     2982293152570     2963489916398 +0.63 % │     4726059821990     4745687016704 -0.41 % │     990892     989300  +0.16 % │    0    0  +nan % │
├────────────────────────────┼─────────────────────────┼─────────────────────────────────────────────┼─────────────────────────────────────────────┼────────────────────────────────┼───────────────────┤
│             coq-coquelicot │   69.74   69.21 +0.77 % │      190595122317      189749401304 +0.45 % │      242969620594      244185341792 -0.50 % │     584068     584272  -0.03 % │    0    0  +nan % │
├────────────────────────────┼─────────────────────────┼─────────────────────────────────────────────┼─────────────────────────────────────────────┼────────────────────────────────┼───────────────────┤
│              coq-fiat-core │  105.13  104.16 +0.93 % │      299687409941      299170628414 +0.17 % │      397419042154      397837921887 -0.11 % │     488020     486720  +0.27 % │    0    0  +nan % │
└────────────────────────────┴─────────────────────────┴─────────────────────────────────────────────┴─────────────────────────────────────────────┴────────────────────────────────┴───────────────────┘
```

Strangely enough, no time difference but quite relevant memory consumption difference though. I'll try with a few tweaks.

**Comment by ppedrot** [comment]
Not conclusive.
https://gitlab.com/coq/coq/-/jobs/757529732
```
┌─────────────────────────────┬─────────────────────────┬─────────────────────────────────────────────┬─────────────────────────────────────────────┬────────────────────────────────┬───────────────────┐
│                             │      user time [s]      │                 CPU cycles                  │              CPU instructions               │     max resident mem [KB]      │    mem faults     │
│                             │                         │                                             │                                             │                                │                   │
│                package_name │     NEW     OLD PDIFF   │               NEW               OLD PDIFF   │               NEW               OLD PDIFF   │        NEW        OLD  PDIFF   │  NEW  OLD PDIFF   │
├─────────────────────────────┼─────────────────────────┼─────────────────────────────────────────────┼─────────────────────────────────────────────┼────────────────────────────────┼───────────────────┤
│              coq-coquelicot │   69.47   69.89 -0.60 % │      190568048441      190865282846 -0.16 % │      243577505475      245086546990 -0.62 % │     584516     584576  -0.01 % │    0    0  +nan % │
├─────────────────────────────┼─────────────────────────┼─────────────────────────────────────────────┼─────────────────────────────────────────────┼────────────────────────────────┼───────────────────┤
│                   coq-verdi │  121.00  121.67 -0.55 % │      334489759963      335174829330 -0.20 % │      410623350607      413199299360 -0.62 % │     576008     576144  -0.02 % │    0    0  +nan % │
├─────────────────────────────┼─────────────────────────┼─────────────────────────────────────────────┼─────────────────────────────────────────────┼────────────────────────────────┼───────────────────┤
│                coq-compcert │  658.11  661.63 -0.53 % │     1834478480497     1842656857154 -0.44 % │     2535334303372     2553670330083 -0.72 % │    1129876    1130240  -0.03 % │    0    0  +nan % │
├─────────────────────────────┼─────────────────────────┼─────────────────────────────────────────────┼─────────────────────────────────────────────┼────────────────────────────────┼───────────────────┤
│        coq-mathcomp-algebra │  139.52  139.99 -0.34 % │      388511836899      389003259645 -0.13 % │      499404061631      502544089584 -0.62 % │     562980     564124  -0.20 % │    0    0  +nan % │
├─────────────────────────────┼─────────────────────────┼─────────────────────────────────────────────┼─────────────────────────────────────────────┼────────────────────────────────┼───────────────────┤
│                coq-rewriter │  606.62  608.64 -0.33 % │     1682323765854     1688535353316 -0.37 % │     2570706354620     2578771654220 -0.31 % │     989916    1031920  -4.07 % │    0    0  +nan % │
├─────────────────────────────┼─────────────────────────┼─────────────────────────────────────────────┼─────────────────────────────────────────────┼────────────────────────────────┼───────────────────┤
│      coq-mathcomp-character │  159.44  159.92 -0.30 % │      443763747775      444755718464 -0.22 % │      612690531633      617823149302 -0.83 % │     718928     719116  -0.03 % │    0    0  +nan % │
├─────────────────────────────┼─────────────────────────┼─────────────────────────────────────────────┼─────────────────────────────────────────────┼────────────────────────────────┼───────────────────┤
│             coq-lambda-rust │ 1254.42 1256.70 -0.18 % │     3490204577762     3496201568681 -0.17 % │     4864246977685     4937969094353 -1.49 % │    1166976    1164360  +0.22 % │    0    0  +nan % │
├─────────────────────────────┼─────────────────────────┼─────────────────────────────────────────────┼─────────────────────────────────────────────┼────────────────────────────────┼───────────────────┤
│      coq-mathcomp-ssreflect │   50.19   50.27 -0.16 % │      137486168532      137494180314 -0.01 % │      174839792902      176083580528 -0.71 % │     523704     521408  +0.44 % │    0    0  +nan % │
├─────────────────────────────┼─────────────────────────┼─────────────────────────────────────────────┼─────────────────────────────────────────────┼────────────────────────────────┼───────────────────┤
│            coq-math-classes │  188.97  189.17 -0.11 % │      527585048953      526900367609 +0.13 % │      696280356339      697609858115 -0.19 % │     504452     504804  -0.07 % │    0    0  +nan % │
├─────────────────────────────┼─────────────────────────┼─────────────────────────────────────────────┼─────────────────────────────────────────────┼────────────────────────────────┼───────────────────┤
│             coq-fiat-crypto │ 4972.92 4977.04 -0.08 % │    13813810511159    13825552689465 -0.08 % │    23850135096692    23989912964485 -0.58 % │    2400332    2437280  -1.52 % │    0    0  +nan % │
├─────────────────────────────┼─────────────────────────┼─────────────────────────────────────────────┼─────────────────────────────────────────────┼────────────────────────────────┼───────────────────┤
│               coq-fourcolor │ 2578.45 2580.20 -0.07 % │     7188901679873     7191512660016 -0.04 % │    11836165941563    11842168720857 -0.05 % │     774752     774760  -0.00 % │    0    0  +nan % │
├─────────────────────────────┼─────────────────────────┼─────────────────────────────────────────────┼─────────────────────────────────────────────┼────────────────────────────────┼───────────────────┤
│       coq-mathcomp-solvable │  171.55  171.62 -0.04 % │      477064201287      477683299194 -0.13 % │      669527611253      676027678848 -0.96 % │     691632     691668  -0.01 % │    0    0  +nan % │
├─────────────────────────────┼─────────────────────────┼─────────────────────────────────────────────┼─────────────────────────────────────────────┼────────────────────────────────┼───────────────────┤
│                         coq │  528.06  528.27 -0.04 % │     1476328739242     1476338562199 -0.00 % │     1986486983990     1995263757160 -0.44 % │     591732     596756  -0.84 % │    0    0  +nan % │
├─────────────────────────────┼─────────────────────────┼─────────────────────────────────────────────┼─────────────────────────────────────────────┼────────────────────────────────┼───────────────────┤
│            coq-fiat-parsers │  648.04  648.29 -0.04 % │     1812312601544     1814697444709 -0.13 % │     2715989074130     2738397049236 -0.82 % │    2997480    2996900  +0.02 % │    0    0  +nan % │
├─────────────────────────────┼─────────────────────────┼─────────────────────────────────────────────┼─────────────────────────────────────────────┼────────────────────────────────┼───────────────────┤
│                   coq-color │  532.89  532.90 -0.00 % │     1489684835144     1487747270469 +0.13 % │     1877697341674     1895782907179 -0.95 % │    1402824    1421840  -1.34 % │    0    0  +nan % │
├─────────────────────────────┼─────────────────────────┼─────────────────────────────────────────────┼─────────────────────────────────────────────┼────────────────────────────────┼───────────────────┤
│                  coq-geocoq │ 1466.23 1465.88 +0.02 % │     4084528999983     4084756909964 -0.01 % │     6201358613791     6214911038380 -0.22 % │    1157408    1149504  +0.69 % │    0    0  +nan % │
├─────────────────────────────┼─────────────────────────┼─────────────────────────────────────────────┼─────────────────────────────────────────────┼────────────────────────────────┼───────────────────┤
│                 coq-unimath │ 4003.04 4000.70 +0.06 % │    11137577039405    11126117660273 +0.10 % │    20953314382338    21166488685740 -1.01 % │    1116976    1113232  +0.34 % │    0    0  +nan % │
├─────────────────────────────┼─────────────────────────┼─────────────────────────────────────────────┼─────────────────────────────────────────────┼────────────────────────────────┼───────────────────┤
│               coq-fiat-core │  105.48  105.36 +0.11 % │      300733170780      300216268298 +0.17 % │      399461104420      400352958047 -0.22 % │     487540     486668  +0.18 % │    0    0  +nan % │
├─────────────────────────────┼─────────────────────────┼─────────────────────────────────────────────┼─────────────────────────────────────────────┼────────────────────────────────┼───────────────────┤
│                coq-coqprime │   80.64   80.48 +0.20 % │      221690542038      221187470256 +0.23 % │      309583296407      311357974878 -0.57 % │     767928     768192  -0.03 % │    0    0  +nan % │
├─────────────────────────────┼─────────────────────────┼─────────────────────────────────────────────┼─────────────────────────────────────────────┼────────────────────────────────┼───────────────────┤
│              coq-verdi-raft │ 1257.50 1253.78 +0.30 % │     3498842905594     3487280310138 +0.33 % │     4874913987316     4890780775919 -0.32 % │     934204     932904  +0.14 % │    0    0  +nan % │
├─────────────────────────────┼─────────────────────────┼─────────────────────────────────────────────┼─────────────────────────────────────────────┼────────────────────────────────┼───────────────────┤
│  coq-performance-tests-lite │  546.58  544.73 +0.34 % │     1527856997201     1520513990570 +0.48 % │     2798870583657     2794511702746 +0.16 % │     768424     751112  +2.30 % │    0    0  +nan % │
├─────────────────────────────┼─────────────────────────┼─────────────────────────────────────────────┼─────────────────────────────────────────────┼────────────────────────────────┼───────────────────┤
│          coq-mathcomp-field │  291.34  290.35 +0.34 % │      811051561343      808830629668 +0.27 % │     1242160136734     1247654117511 -0.44 % │     804856     812088  -0.89 % │    0    0  +nan % │
├─────────────────────────────┼─────────────────────────┼─────────────────────────────────────────────┼─────────────────────────────────────────────┼────────────────────────────────┼───────────────────┤
│       coq-mathcomp-fingroup │   47.88   47.71 +0.36 % │      132531435254      132694163876 -0.12 % │      182078271420      183691446567 -0.88 % │     486168     486132  +0.01 % │    0    0  +nan % │
├─────────────────────────────┼─────────────────────────┼─────────────────────────────────────────────┼─────────────────────────────────────────────┼────────────────────────────────┼───────────────────┤
│       coq-engine-bench-lite │  382.33  380.87 +0.38 % │     1074865045608     1070485860078 +0.41 % │     1838141328327     1838788939786 -0.04 % │    3507460    3507232  +0.01 % │    0    0  +nan % │
├─────────────────────────────┼─────────────────────────┼─────────────────────────────────────────────┼─────────────────────────────────────────────┼────────────────────────────────┼───────────────────┤
│ coq-rewriter-perf-SuperFast │ 1062.35 1058.27 +0.39 % │     2954960975822     2944968711318 +0.34 % │     4641637050537     4641907391799 -0.01 % │    1070876    1070696  +0.02 % │    0    0  +nan % │
├─────────────────────────────┼─────────────────────────┼─────────────────────────────────────────────┼─────────────────────────────────────────────┼────────────────────────────────┼───────────────────┤
│                    coq-hott │  324.32  322.94 +0.43 % │      876302058800      876913981736 -0.07 % │     1412467669948     1427620185698 -1.06 % │     634188     734040 -13.60 % │    0    0  +nan % │
├─────────────────────────────┼─────────────────────────┼─────────────────────────────────────────────┼─────────────────────────────────────────────┼────────────────────────────────┼───────────────────┤
│      coq-mathcomp-odd-order │ 1068.30 1063.24 +0.48 % │     2973773364170     2962246548532 +0.39 % │     4725439350362     4745714532668 -0.43 % │     989372     989312  +0.01 % │    0    0  +nan % │
├─────────────────────────────┼─────────────────────────┼─────────────────────────────────────────────┼─────────────────────────────────────────────┼────────────────────────────────┼───────────────────┤
│                coq-bedrock2 │  224.15  222.98 +0.52 % │      619379301695      616662459671 +0.44 % │     1017547286244     1020579080853 -0.30 % │    1087056    1086900  +0.01 % │    0    0  +nan % │
├─────────────────────────────┼─────────────────────────┼─────────────────────────────────────────────┼─────────────────────────────────────────────┼────────────────────────────────┼───────────────────┤
│                 coq-bignums │   57.63   57.26 +0.65 % │      159318301287      158977650428 +0.21 % │      214701088890      215195571961 -0.23 % │     473144     471772  +0.29 % │    0    0  +nan % │
├─────────────────────────────┼─────────────────────────┼─────────────────────────────────────────────┼─────────────────────────────────────────────┼────────────────────────────────┼───────────────────┤
│                   coq-flocq │  159.24  158.02 +0.77 % │      441387137813      439635389818 +0.40 % │      548192433387      551371988550 -0.58 % │     822284     821840  +0.05 % │    0    0  +nan % │
├─────────────────────────────┼─────────────────────────┼─────────────────────────────────────────────┼─────────────────────────────────────────────┼────────────────────────────────┼───────────────────┤
│                    coq-corn │ 1510.56 1496.39 +0.95 % │     4202404452235     4169234096876 +0.80 % │     6369787929547     6395624530579 -0.40 % │     888328     831120  +6.88 % │    0    0  +nan % │
├─────────────────────────────┼─────────────────────────┼─────────────────────────────────────────────┼─────────────────────────────────────────────┼────────────────────────────────┼───────────────────┤
│                 coq-coqutil │   59.75   58.93 +1.39 % │      165404386531      163675097838 +1.06 % │      204561875809      204875715185 -0.15 % │     531764     530880  +0.17 % │    0    0  +nan % │
└─────────────────────────────┴─────────────────────────┴─────────────────────────────────────────────┴─────────────────────────────────────────────┴────────────────────────────────┴───────────────────┘
```

**Pull request closed without merging**